build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+plugins {
+    id("java-library")
+    id("maven-publish")
+
+    val th2PluginVersion = "0.0.4"
+    id("com.exactpro.th2.gradle.base") version th2PluginVersion
```

Comment:
note: gRPC th2 plugin automatically applies `base` plugins if it is applied to the root project. You can remove explicit application here

Reply:
But this is the root project

Reply:
Yes, and this is why you don't need to add `base` plugin manually. You can apply `grpc` and `publish` plugins only